components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -1057,14 +1059,6 @@ const string kPolygonZKEVMChainId = "0x44d";
 const string kZkSyncEraChainId = "0x144";
 const string kBaseMainnetChainId = "0x2105";
 
```

Comment:
Could we postpone wallet's mojom refactoring and keep it as a single file by now?
I agree it should be split somehow sooner rather than later, but as a separate PR.

Reply:
What is the reason of postponing it? The single file has been bloated in a very bad way and we should split it up when we get a chance.

Reply:
sigh I found it very hard to rebase the `AccountId` stuff because it will needs to be pulled out to base.mojom and many stuff `AccountId` depends on due to `SignMessageRequest` now contains `AccountId`
I will pull out the split commit

Reply:
rebased and removed